### ISSUE TITLE
tctl auth sign - database access

### ISSUE THREAD

**Issue by scottpgallagher** [gravitational/teleport#12910]
## Details
Describe the documentation improvements you wish to see.
When users setup database access they need to export a certificate to be used. They need to run a `tctl auth sign` command, but without the right permissions users can't perform this command. While this command can be run on the Auth server Teleport Cloud users don't have access to that system.

Teleport should document the permissions needed so a user can run this command locally:

```
spec:
  allow:
    db_labels:
      '*': '*'
    impersonate:
      roles:
      - Db
      users:
      - Db
```

### Category

- Improve Existing

**Comment by zmb3** [comment]
This is documented, but it's a bit hard to find:

https://goteleport.com/docs/database-access/guides/postgres-self-hosted/?scope=cloud#step-25-create-a-certificatekey-pair

**Comment by zmb3** [comment]
@GavinFrazar anything left to do here or do you think we can close this?

**Comment by GavinFrazar** [comment]
should be resolved by #13902 and https://github.com/gravitational/teleport/pull/14528
